js/src/components/google-combo-account-card/connect-ads/connect-cta.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import AppButton from '.~/components/app-button';
+import ConnectedIconLabel from '.~/components/connected-icon-label';
+import useEventPropertiesFilter from '.~/hooks/useEventPropertiesFilter';
+import { FILTER_ONBOARDING } from '.~/utils/tracks';
+
+/**
+ * Connect CTA component.
+ *
+ * @param {Object} props Props.
+ * @param {boolean} props.isConnected Whether the account is connected.
+ * @param {number} props.id Account ID.
+ * @param {Function} props.handleConnectClick Callback to handle the connect click.
+ * @param {string} props.value Connected account ID.
+ *
+ * @return {JSX.Element} Connect CTA component.
+ */
+const ConnectCTA = ( { isConnected, id, handleConnectClick, value } ) => {
+	const getEventProps = useEventPropertiesFilter( FILTER_ONBOARDING );
+
+	if ( isConnected && id === Number( value ) ) {
```

Comment:
@ankitrox I don't think we need to check for `id === Number( value )` here because if the account is connected, the select should be read only. 
https://www.figma.com/design/fqR0EHi63lWahRcVTKCcba/Google-Listings-%26-Ads-v2.x?node-id=7662-6154&node-type=frame&t=X7RxteEeRgucqbnq-0

Meaning the only way to connect to another Ads account will be the user clicking the "Or, connect to another" . See conversation here:
https://github.com/woocommerce/google-listings-and-ads/issues/2603#issuecomment-2372873698

Existing implementation can be found here: `js/src/components/google-ads-account-card/disconnect-account.js` Maybe we can even re use the `DisconnectAccount` component.

Reply:
@asvinb I think we will need this check. Please refer [this design](https://www.figma.com/design/fqR0EHi63lWahRcVTKCcba/Google-Listings-%26-Ads-v2.x?node-id=7662-6154&node-type=frame&t=1KlDSTB27u712REa-0) in figma. If we click the `Edit` button, it expands both Existing MC and Existing Ads account cards where we would need to show the connected accounts.

Reply:
@ankitrox Not sure if you forgot to push your changes after my comment . https://github.com/woocommerce/google-listings-and-ads/pull/2640#discussion_r1799425508
We need to do the following:
* In the connected state, we should disable the select as per the [designs](https://www.figma.com/design/fqR0EHi63lWahRcVTKCcba/Google-Listings-%26-Ads-v2.x?node-id=7662-6154&node-type=frame&t=FZW0ZlwmMV6RKllP-0).
* Add the "Or, connect to a different Google Ads account" button when in connected state. Check the [conversation here](https://github.com/woocommerce/google-listings-and-ads/issues/2603#issuecomment-2372873698).

Reply:
<img width="775" alt="Screenshot 2024-10-18 at 10 18 28 AM" src="https://github.com/user-attachments/assets/578422d5-642c-403c-ad7e-07f1871f1567">

1. Select is in disabled state, meaning only one option is present.
2. `'Or, connect to a different Google Ads account'` text is present when in Connected state.